### ISSUE TITLE
Consider removing Buffer from node code, replace with native array type

### ISSUE THREAD

**Issue by ericblade** [ericblade/quagga2#538]
More info: https://sindresorhus.com/blog/goodbye-nodejs-buffer

Also would provide a better fix for: https://github.com/ericblade/quagga2/issues/509

The current fix is just to install the type, because it doesn't matter... but it could matter in user code.

**Comment by yogeshgadge** [comment]
added PR https://github.com/ericblade/quagga2/pull/539

**Comment by yogeshgadge** [comment]
side observations:  root tsconfig.json has types ["mocha", "node"]
probably consider tsconfig.test.json for the mocha tests inside src/

**Comment by ericblade** [comment]
good idea.  there's been some recent upgrades to typescript's handling of nested tsconfig files, too, an update to current ts is probably a good idea too

**Comment by ericblade** [comment]
not sure why this didn't auto close when we merged it .. so closing now. thakns!
